packages/circuit-ui/styles/style-mixins.ts

```diff
@@ -309,3 +308,44 @@ export const inputOutline = (
     }
   `;
 };
+
+/**
+ * Visually communicates that the listItem (eg. Popover or Dropdown component) is hovered, active or focused.
+ */
+export const listItem = (
```

Comment:
In which component are we thinking of using this besides the Popover? If we don't have plans of reusing I'd keep it in the PopoverItem component instead of a mixin (but I can be convinced otherwise)

Reply:
@amelako and I put this here so it could be reused in the ListItem component in the future (initially, that was planned for this quarter).